jobs/dynatrace-oneagent/templates/pre-start.erb

```diff
@@ -96,6 +97,14 @@ setHostTags() {
     echo -n "${HOST_TAGS}" > "${hostTagsFile}"
 }
 
+setHostProps() {
+    local hostPropsFile="${CONFIG_DIR}/hostcustomproperties.conf"
+
+    # We need to save the file content even if it's empty.
```

Comment:
Same for the linux part. Why not just check for HOST_PROPS to avoid writing a pointless file?